modules/storage/storage-account/blobServices/containers/deploy.bicep

```diff
@@ -0,0 +1,73 @@
+@maxLength(24)
+@description('Required. Name of the Storage Account.')
+param storageAccountName string
+
+@description('Optional. Name of the blob service.')
+param blobServicesName string = 'default'
+
+@description('Required. The name of the storage container to deploy')
+param name string
+
+@description('Optional. Name of the immutable policy.')
+param immutabilityPolicyName string = 'default'
+
+@allowed([
+  'Container'
+  'Blob'
+  'None'
+])
+@description('Optional. Specifies whether data in the container may be accessed publicly and the level of access.')
+param publicAccess string = 'None'
+
+@description('Optional. Configure immutability policy.')
```

Comment:
I agree. I guess you mean like in the original [documentation](https://github.com/Azure/ResourceModules/tree/main/arm/Microsoft.Storage/storageAccounts#parameter-usage-roleassignments)?

Example:
### Parameter Usage: `roleAssignments`

Create a role assignment for the given resource. If you want to assign a service principal / managed identity that is created in the same deployment, make sure to also specify the `'principalType'` parameter and set it to `'ServicePrincipal'`. This will ensure the role assignment waits for the principal's propagation in Azure.

```json
"roleAssignments": {
    "value": [
        {
            "roleDefinitionIdOrName": "Reader",
            "description": "Reader Role Assignment",
            "principalIds": [
                "12345678-1234-1234-1234-123456789012", // object 1
                "78945612-1234-1234-1234-123456789012" // object 2
            ]
        },
        {
            "roleDefinitionIdOrName": "/providers/Microsoft.Authorization/roleDefinitions/c2f4ef07-c644-48eb-af81-4b1b4947fb11",
            "principalIds": [
                "12345678-1234-1234-1234-123456789012" // object 1
            ],
            "principalType": "ServicePrincipal"
        }
    ]
}
```